lighthouse-core/lib/page-functions.js

```diff
@@ -168,6 +168,16 @@ function getOuterHTMLSnippet(element, ignoreAttrs = [], snippetCharacterLimit =
   }
 }
 
+/**
+ * Get the maximum size of a texture the GPU can handle
+ * @see https://bugs.chromium.org/p/chromium/issues/detail?id=770769#c13
+ */
+/* istanbul ignore next */ // todo, c8 ignore?
+function getMaxTextureSize() {
+  const canvas = document.createElement('canvas');
+  const webGL = canvas.getContext('webgl');
```

Comment:
tsc would correctly call out `webGL` as `WebGLRenderingContext | null` here if the file wasn't `// @ts-nocheck`, so need to handle that case (e.g. webgl is disabled on that machine, as discussed in the other comments :)